subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/NotificationService.kt

```diff
@@ -54,21 +54,25 @@ class NotificationService(
             subscriptionId = subscription.id,
             data = buildNotifData(entity, subscription.notification)
         )
-
-        if (subscription.notification.endpoint.uri.toString() == "embedded-firebase") {
+        val uri = subscription.notification.endpoint.uri.toString()
+        logger.info("Notification is about to be sent to $uri")
+        if (uri == "embedded-firebase") {
             val fcmDeviceToken = subscription.notification.endpoint.getInfoValue("deviceToken")
             return callFCMSubscriber(entityId, subscription, notification, fcmDeviceToken)
         } else {
             var request =
-                WebClient.create(subscription.notification.endpoint.uri.toString()).post() as WebClient.RequestBodySpec
+                WebClient.create(uri).post() as WebClient.RequestBodySpec
             subscription.notification.endpoint.info?.forEach {
                 request = request.header(it.key, it.value)
             }
             return request
                 .bodyValue(notification)
                 .exchange()
+                .doOnError { e -> logger.error("Failed to send notification to $uri : ${e.message}") }
                 .map {
-                    Triple(subscription, notification, it.statusCode() == HttpStatus.OK)
+                    val success = it.statusCode() == HttpStatus.OK
+                    logger.info("The notification sent has been received with ${if (success) "success" else "failure"}")
```

Comment:
Is it supposed to be logged as an info when it fails ?

Reply:
The error would be the response of the subscriber, not an error from Stellio. The error on the execution request is already logged in doOnError method.